src/i18n.js

```diff
@@ -0,0 +1,36 @@
+import i18n from 'i18next';
```

Comment:
I think this file still needs to be imported in index.js

https://github.com/i18next/i18next-http-backend

Reply:
I imported the i18n.js file into index.js
I'm not sure if that's all